core/src/autogluon/core/trainer/abstract_trainer.py

```diff
@@ -540,21 +546,17 @@ def stack_new_level(
             **core_kwargs,
         )
 
-        if X_val is None:
-            aux_models = self.stack_new_level_aux(
-                X=X, y=y, base_model_names=core_models, level=level + 1, infer_limit=infer_limit, infer_limit_batch_size=infer_limit_batch_size, **aux_kwargs
-            )
-        else:
-            aux_models = self.stack_new_level_aux(
-                X=X_val,
-                y=y_val,
-                fit=False,
-                base_model_names=core_models,
-                level=level + 1,
-                infer_limit=infer_limit,
-                infer_limit_batch_size=infer_limit_batch_size,
-                **aux_kwargs,
-            )
+        aux_models = []
+        if full_weighted_ensemble:
+            full_aux_kwargs = aux_kwargs.copy()
+            if additional_full_weighted_ensemble:
+                full_aux_kwargs["name_extra"] = "_FULL"
```

Comment:
make sure this isn't "_FULL" to avoid name clash with `predictor.refit_full()`

An alternative could be "_ALL"